vocabularies/Offline.xml

```diff
@@ -49,7 +49,20 @@ Please refer to the [Offline OData product home](https://help.sap.com/doc/f53c64
            attempt to upload any changes that are made to them locally.
            The service as implemented on the server is described by the CSDL document if
            all elements tagged as `Offline.ClientOnly` are removed together with all subsequently dangling references.</String>
-        </Annotation>        
+        </Annotation>
+      </Term>
+
+      <Term Name="LocalDraft" Type="Core.Tag" DefaultValue="true" Nullable="false" AppliesTo="EntitySet">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Marks a local draft version of an entity set, which uses the same entity type as a non-draft entity set defined in the backend defined metadata."/>
```

Comment:
If two backend-defined entity sets use the same entity type and a third entity set tagged as `Offline.LocalDraft` uses the same entity type, it is unclear to which of the two other entity sets it corresponds. Should `Offline.LocalDraft` perhaps contain the name of the backend-defined entity set rather than being a boolean?

Reply:
The API we are providing for clients when "activating" a draft allows the client to indicate the "active" entity set into which the draft will be activated (only necessary to indicate this if there is an ambiguity).

However your question/suggestion is a good one, perhaps we can change it to Type="Edm.String" Nullable="true" without a default value (the implicit default of null can then indicate that the draft set is linked with the only other entity set using the same entity type).

Reply:
The term is now change to <Term Name="LocalDraft" Type="Edm.String" Nullable="true" AppliesTo="EntitySet">
thanks a lot for the suggestions.

Reply:
I like it